lib/mpl_toolkits/mplot3d/axis3d.py

```diff
@@ -178,23 +178,74 @@ def get_rotate_label(self, text):
             return len(text) > 4
 
     def _get_coord_info(self, renderer):
-        mins, maxs = np.array([
-            self.axes.get_xbound(),
-            self.axes.get_ybound(),
-            self.axes.get_zbound(),
-        ]).T
-        centers = (maxs + mins) / 2.
-        deltas = (maxs - mins) / 12.
-        mins = mins - deltas / 4.
-        maxs = maxs + deltas / 4.
-
-        vals = mins[0], maxs[0], mins[1], maxs[1], mins[2], maxs[2]
-        tc = self.axes.tunit_cube(vals, self.axes.M)
-        avgz = [tc[p1][2] + tc[p2][2] + tc[p3][2] + tc[p4][2]
-                for p1, p2, p3, p4 in self._PLANES]
-        highs = np.array([avgz[2*i] < avgz[2*i+1] for i in range(3)])
-
-        return mins, maxs, centers, deltas, tc, highs
+        mins, maxs = np.array(
+            [
+                self.axes.get_xbound(),
+                self.axes.get_ybound(),
+                self.axes.get_zbound(),
+            ]
+        ).T
```

Comment:
It's just the format `black` suggests.

Reply:
please revert any un-related changes.  We don't use `black` and its distracting to deal with fundamental changes and drive-by formatting changes ;-)

Reply:
The entire _get_coord_info() is a drive-by change that I had to do to figure out what is going on, it doesn't do much besides making it easier (for me atleast...) to read. 
Shall I make a new PR or do you prefer this function to remain uncommented?

Reply:
Please move pure stylistic/readability changes to a separate PR, if they are not in context of code you have to modify for functionality. The logic of changing the vertical axis is involved. It will help with later debugging if we have a PR saying "cleanup/style only; no functional changes" and a second smaller one that implements the new logic.

Reply:
See #20242 for the `_get_coord_info` changes.